src/apps/openfluid-devstudio/WorkspaceToolbar.cpp

```diff
@@ -0,0 +1,194 @@
+/*
+
+  This file is part of OpenFLUID software
+  Copyright(c) 2007, INRA - Montpellier SupAgro
+
+
+ == GNU General Public License Usage ==
+
+  OpenFLUID is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OpenFLUID is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OpenFLUID. If not, see <http://www.gnu.org/licenses/>.
+
+
+ == Other Usage ==
+
+  Other Usage means a use of OpenFLUID that is inconsistent with the GPL
+  license, and requires a written agreement between You and INRA.
+  Licensees for Other Usage of OpenFLUID may use this file in accordance
+  with the terms contained in the written agreement between You and INRA.
+
+ */
+
+
+/**
+ @file WorkspaceToolbar.cpp
+
+ @author Aline LIBRES <aline.libres@gmail.com>
+ @author Jean-Christophe Fabre <jean-christophe.fabre@inra.fr>
+ @author Armel Thöni <armel.thoni@inrae.fr>
+*/
+
+
+#include <QMenu>
+#include <QToolButton>
+#include <QPushButton>
+#include <QWidget>
+#include <QLabel>
+#include <QCheckBox>
+#include <QComboBox>
+#include <QVBoxLayout>
+#include <QHBoxLayout>
+
+#include <apps/openfluid-devstudio/WorkspaceToolbar.hpp>
```

Comment:
`<apps/openfluid-devstudio/WorkspaceToolbar.hpp>` is not in the OpenFLUID framework, use `"WorkspaceToolbar.hpp"` instead for inclusion, then put it after inclusions ofstd/frameworks headers
see https://community.openfluid-project.org/scidev/codestyle/#included-header-files